packages/create-waku/src/tests/cli-with-args.test.ts

```diff
@@ -83,6 +83,20 @@ describe('create-waku CLI with args', () => {
     expect(stdout).toContain('Choose a starter template');
   });
 
+  test('choosing an option chooses that option', async () => {
+    const cmd = execa({
+      cwd: import.meta.dirname, timeout: 2000, reject: false
+    })`node ${CLI_PATH} --project-name ${projectName} --choose --skip-install`
```

Comment:
would it make sense to define the executor at the top of this file after the imports so that it can be reused/read a lil easier?

```suggestion
const executor = execa({
  cwd: import.meta.dirname, timeout: 2000, reject: false
})
    //...
    
    const cmd = executor`node ${CLI_PATH} --project-name ${projectName} --choose --skip-install`
```